lib/metanorma/cli/collection.rb

```diff
@@ -26,8 +27,14 @@ def collection_file
         @collection_file ||= Metanorma::Collection.parse(file)
       end
 
+      def verify_required_options_are_present
+        if collection_options[:output_folder].nil?
+          raise ArgumentError.new("Missing a required option '--output-folder'")
```

Comment:
We shoud provide a default path for "output folder" just like how `metanorma compile sources/sample.adoc` uses the location of the input file as the output folder:

* `metanorma compile sources/sample.adoc` => `sources/sample.xml` output
* `metanorma collection sources/collection.yaml` => `sources/collection.xml` output

Reply:
That's a good point to set the source path as a default output folder 👍🏼 

Actually, after looking into it I came across a bit tricky issue - looks like we are recreating the output folder before we write the outputs, so if we use the `source directory` as the output path then it gets removed (along with the source file) in the recreation phase - [here is the source code](https://github.com/metanorma/metanorma/blob/743fbbc7c382aae4a8cf28da3690fd0260e846c1/lib/metanorma/collection_renderer.rb#L47-L51).

So, what do you guys think about using `...source/output` as a default output directory if nothing was provided?

// @ronaldtse, @opoudjis

Reply:
That code is clearly wrong, we should NOT be removing the output directory in any case.

We want to use the source path as the default output folder.

Reply:
I'm not sure about the exact reasoning whey we need to remove the output directory before writing the output. @andrew2net, @opoudjis - Do you guys have any idea why we are doing that, do [we still need to keep that logic there](https://github.com/metanorma/metanorma/blob/743fbbc7c382aae4a8cf28da3690fd0260e846c1/lib/metanorma/collection_renderer.rb#L47-L51)?

Reply:
I would have done it to ensure that only data from the current run was present in the folder. Arguably that's overly cautious. Maybe. Ok, get rid of it.

Reply:
Cool, I just created a PR for that - https://github.com/metanorma/metanorma/pull/259, once that's merged and released then we can merge this one. FYI, I will update this PR to use the source folder next :)